src/Gui/NavigationStyle.cpp

```diff
@@ -114,33 +115,47 @@ class FCSphereSheetProjector : public SbSphereSheetProjector {
         SbRotation rot = inherited::getRotation(point1, point2);
         if (orbit == Trackball)
             return rot;
-
-        // 0000333: Turntable camera rotation
-        SbVec3f axis;
-        float angle;
-        rot.getValue(axis, angle);
-        SbVec3f dif = point1 - point2;
-        if (fabs(dif[1]) > fabs(dif[0])) {
-            SbVec3f xaxis(1,0,0);
-            if (dif[1] < 0)
-                angle = -angle;
-            rot.setValue(xaxis, angle);
-        }
-        else {
-            SbVec3f zaxis(0,0,1);
-            this->worldToScreen.multDirMatrix(zaxis, zaxis);
-            if (zaxis[1] < 0) {
-                if (dif[0] < 0)
+        else if (orbit == Turntable) {
+            // 0000333: Turntable camera rotation
```

Comment:
What does the 0000333 mean? I know this is not your code but maybe you have an idea?

Reply:
I don't know, it was there before so I didn't want to touch it.

Reply:
That's the old.ticket numbering format for the mantis bug tracker. It's linking to ticket: https://tracker.freecadweb.org/view.php?id=333

Reply:
Oh great, so now that we have both possibilities, should we remove this comment ?